.github/workflows/dbt_slim_ci.yml

```diff
@@ -59,3 +59,8 @@ jobs:
       - name: dbt test incremental model(s) if applicable
         if: env.INC_MODEL_COUNT > 0
         run: "dbt test --select state:modified,config.materialized:incremental --defer --state ."
+
+      - name: Run DuneSQL Check
+        if: env.INC_MODEL_COUNT > 0
```

Comment:
i don't think this is needed here. i'd imagine you want to run on all spells, regardless of materialization type? tables, views and incremental